linear_regression.go

```diff
@@ -8,11 +8,7 @@ package mlpack
 */
```

Comment:
I can't comment on the binary file, but this is the next file. :)  I think that `libmlpack.so.3.3` shouldn't be included. :+1:

Reply:
Removed this file.